sdk/communication/communication-administration/src/common/models.ts

```diff
@@ -12,4 +12,4 @@ export type WithResponse<T> = T & { _response: HttpResponse };
 /**
  * Represents a generic HTTP response
  */
-export type VoidResponse = WithResponse<{}>;
+export type VoidResponse = WithResponse<Record<string, unknown>>;
```

Comment:
```suggestion
export type VoidResponse = WithResponse<Record<string, never>>;
```

Reply:
Josh showed I was wrong. A `&` with a `Record<string, never>` always resolves to `Record<string, never>` which makes sense.